docs/pages/cheatsheet.adoc

```diff
@@ -323,47 +328,162 @@ No syntax change.
 }
 ----
 
+*Example:*
+
+[source,jsl]
+----
+constructor {
+	self.name = "";
+}
+----
+
+////
+//For later use 
+=== Transfer initializer
+
+The initializer is automatically called after the constructor is called or after the transfer object fetching. It can be used to set the values of the transient fields.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**initializer**# [purple]#**;**#
+----
+////
+
+=== Transfer create
+
+The create operation is automatically called when a new transfer object is saved for the first time. It can be used to create the mapped entity.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**create**# &#x5B;[purple]##**(**##&lt;parameter>[purple]##**)**##] [purple]#**;**#
+----
+
+=== Transfer update
+
+The update operation is automatically called when a transfer object is saved. It can be used to update the mapped entity.
```

Comment:
> The update operation is automatically called when a transfer object is saved.

Suggestion:

The update operation is called to save changes for an existing transfer object by overriding the default behaviour.

Reply:
done